tests/foreman/ui/test_setting.py

```diff
@@ -556,6 +557,50 @@ def test_negative_update_foreman_url_param(self):
                     self.assertNotEqual(param_value, self.saved_element)
 
     @tier1
+    def test_positive_update_login_page_footer_text(self):
+        """Updates parameter "Login_page_footer_text" under General tab
+
+        :id: 56a983c4-925f-4cbe-8fdb-ce344219d739
+
+        :expectedresults: Parameter is updated successfully
+
+        :CaseImportance: Critical
+        """
+        self.tab_locator = tab_locators['settings.tab_general']
+        self.param_name = 'login_text'
+        with Session(self.browser) as session:
+            self.original_value = self.settings.get_saved_value(
```

Comment:
This field (original_value) is created but never used here. Is it used later, maybe on tear down?